### PR TITLE
feat(notifications): add region to unhealthy

### PR DESCRIPTION
**Pull request by emjburns**
Updating the subject of the notification to have a list of regions

Screen Shot 2020-10-09 at 4 15 28 PM

### CODE REVIEW

**Review by gal-yardeni — APPROVED** [review]
so fancy 🎉

File: keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/UnhealthyNotificationListener.kt [hunk 1]
```diff
@@ -82,7 +82,7 @@ class UnhealthyNotificationListener(
     val resourceUrl = "$spinnakerBaseUrl/#/applications/${resource.application}/clusters?${params.toURL()}"
 
     return Notification(
-      subject = "${resource.spec.displayName} is unhealthy",
+      subject = "${resource.spec.displayName} is unhealthy in ${regions.joinToString(" and ")}",
```
Comment (erikmunson) [inline keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/UnhealthyNotificationListener.kt:85]: I suggest making this a little more human by checking the length of the list and using commas for all the but the last list item if the length exceeds 2. so like`us-east-1, us-west-2, and eu-west-1`. If you have access to the spec here as well, might say "all regions" if the length of this list is equal to the length of the regions in the spec.

it's stuff like this that makes UI codebases _so much fun_...

**Review by erikmunson — APPROVED** [review]
I love this ❤️ — had a suggestion around humanizing the text which is consistent with how we do things in the other parts of the product but either way it's great to see this info without even a click

File: keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/UnhealthyNotificationListener.kt (hunk 1 above)
Comment (emjburns) [reply keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/UnhealthyNotificationListener.kt:85]: Arg. I will attempt this.

**Pull request merged**